lib/meteor/meteorProcessManager.js

```diff
@@ -90,10 +97,29 @@ module.exports = function createMeteorProcessManager (options) {
           env.PORT             = meteorPort;
           env.MONGO_URL        = mongoUrl;
           env.GAGARIN_SETTINGS = "{}"; // only used if METEOR_SETTINGS does not contain gagarin field
+          var main = setup.pathToMain;
+          if (codeFixtures && codeFixtures.length > 0) {
+            var id   = require('shortid').generate();
```

Comment:
@pociej Do we really need this `shortid` here? Why can't we use a simple counter?